assembl/graphql/discussion.py

```diff
@@ -602,6 +607,7 @@ def mutate(root, args, context, info):
         logo = args.get('logo', None)
         with_moderation = args.get('with_moderation', None)
         with_translation = args.get('with_translation', None)
+        with_semantic_analysis = args.get('with_semantic_analysis', None)
```

Comment:
Since with_semantic_analysis is a boolean, why not set default value to False instead of None?